src/priorities/priorities_overlords.ts

```diff
@@ -8,6 +8,8 @@ export var OverlordPriority = {
 	core: {						// Functionality related to spawning more creeps
 		queen  : 100,
 		manager: 101,
+
+		//controllerAttack: 199   // Reserved to give controller attacks a high priority
```

Comment:
controller attackers should have lower priority than defense objectives. put at anything over 299 (warSpawnCutoff)